### PR TITLE
1943 feat action palette

### LINKED ISSUE #1943: [Feature?]: (Docs) Action Palette Reference Page
### Duplicates

- [X] I have searched the existing issues

### Latest version

- [X] I have tested the latest version

### Summary 💡

Create an action palette reference page similar to #1926, which was for the neutral palette. Should be based off of [Figma UI Kit](https://www.figma.com/proto/uJtPfI38D9i8iQg0UGK2E0/Pando-Design-Guidelines?node-id=481-19174&starting-point-node-id=6%3A11626)

### Examples 🌈

`/reference/colors/neutral-palette` in new docs

### Motivation 🔦

Each component of the color palette will need its own page for reference.

### PR DESCRIPTION
**Pull request by kathydlavoie-ps**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Closes #1943

## What is the new behavior?

- Adds action color palette
- moves color display to child component to more reliably render color from tokens (includes changes to neutral-palette for this purpose)
- Updates document status 

## Other information
https://github.com/pluralsight/pando/assets/146388897/70b20d40-4673-4839-b90d-af6c530f8475

### CODE REVIEW

File: docs/src/content/docs/explanation/color.md [hunk 1]
```diff
@@ -19,6 +19,8 @@ A palette for non-actionale items, eg. page content
 
 A palette that represents actionable and interactive items (eg. buttons, text links). Only used for status and feedback.
 
+<a href='/reference/colors/action-palette'>view more about action colors</a>
```
Comment (caseybaggz) [inline docs/src/content/docs/explanation/color.md:22]: There were a few articles back in the day mentioning that "view more" is not accessible in links.

```suggestion
<a href='/reference/colors/action-palette'>View the action color reference docs</a>
```

File: docs/src/content/docs/reference/color/action-palette.mdx [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+  title="background-100-hover"
+  description="Hover state background for default interactive elements"
+  uses="Interactive cards (hover), accordions (hover)"
+  colorTokenName="action.bg.100.hover"
+>
+  <div class={css(colorSwatchStyles, { bg: 'action.bg.100.hover' })} />
+</ColorSwatch>
+
+:::caution[Caution]
+background-100-active is currently being evaluated and is not recommended for use
```
Comment (caseybaggz) [inline docs/src/content/docs/reference/color/action-palette.mdx:156]: I feel like we can add more context here. For example, I don't even know why it is currently being evaluated? If it's that extreme why is it in the palette? These are the questions this statement triggers in my brain.

```suggestion
background-100-active is currently being evaluated and is not recommended for use
```

File: docs/src/content/docs/reference/color/action-palette.mdx [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+<ColorSwatch
+  title="background-100-hover"
+  description="Hover state background for default interactive elements"
+  uses="Interactive cards (hover), accordions (hover)"
+  colorTokenName="action.bg.100.hover"
+>
+  <div class={css(colorSwatchStyles, { bg: 'action.bg.100.hover' })} />
+</ColorSwatch>
+
+:::caution[Caution]
```
Comment (caseybaggz) [inline docs/src/content/docs/reference/color/action-palette.mdx:155]:
```suggestion
:::caution[Beta]
```

File: docs/src/content/docs/reference/color/action-palette.mdx (hunk 2 above)
Comment (kathydlavoie-ps) [reply docs/src/content/docs/reference/color/action-palette.mdx:156]: Agreed. I'll clarify with @bradryanbice 👍

**Review by caseybaggz — APPROVED** [review]
🎉

**Pull request merged**